packages/sitecore-jss-nextjs/src/editing/editing-healthz-middleware.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiResponse, NextApiRequest } from 'next';
```

Comment:
We should make this more generic (not specific to editing). I would remove any "editing" from names/comments and move it out of this submodule.

Reply:
Make sense